be/src/storage/rowset/column_iterator.h

```diff
@@ -100,6 +104,55 @@ class ColumnIterator {
         return Status::NotSupported("ColumnIterator Not Support batch read");
     }
 
+    Status convert_sparse_range_to_io_range(const SparseRange<>& range) {
```

Comment:
put the code inside the implementation class? so that no need to include `io/shared_buffered_input_stream.h` in current header file. It can reduce compile time.